source/includes/_invoice.md

```diff
@@ -4618,455 +2922,63 @@ A template is a document based on [**mustache**](https://mustache.github.io/) th
-            <td>{{invoice.currency}} {{amount}}</td>
-                        
-         </tr>
-                     {{/invoice.invoiceItems}}            
-         <tr>
-                            
-            <td colspan=4 />            
-         </tr>
-                     
-         <tr>
-                            
-            <td colspan=2 />                
-            <td align=right><strong>{{text.invoiceAmount}}</strong></td>
-                            
-            <td align=right><strong>{{invoice.chargedAmount}}</strong></td>
-                        
-         </tr>
-                     
-         <tr>
-                            
-            <td colspan=2 />                
-            <td align=right><strong>{{text.invoiceAmountPaid}}</strong></td>
-                            
-            <td align=right><strong>{{invoice.paidAmount}}</strong></td>
-                        
-         </tr>
-                     
-         <tr>
-                            
-            <td colspan=2 />                
-            <td align=right><strong>{{text.invoiceBalance}}</strong></td>
-                            
-            <td align=right><strong>{{invoice.balance}}</strong></td>
-                        
-         </tr>
-                 
-      </table>
-          
-   </body>
-   
-</html>
-"
+
+`POST http://127.0.0.1:8080/1.0/kb/invoices/manualPayTemplate`
+
+> Example Request:
+
+```shell
+curl -v \
+    -X POST \
+    -u admin:password \
+    -H "X-Killbill-ApiKey: bob" \
+    -H "X-Killbill-ApiSecret: lazar" \
+    -H "Content-Type: text/html" \
+    -H "Accept: text/html" \
+    -H "X-Killbill-CreatedBy: demo" \
+    -H "X-Killbill-Reason: demo" \
+    -H "X-Killbill-Comment: demo" \
+    -d '"Some_HTML_String"' \
+    "http://localhost:8080/1.0/kb/invoices/manualPayTemplate?deleteIfExists=false"	
+```
+
+```java
+String body = "Test HTML String";
+Boolean deleteIfExists = true;
+String template = invoiceApi.uploadInvoiceMPTemplate(body, deleteIfExists,requestOptions);
+
+```
+
+```ruby
+invoice_template = "Some_HTML_String"
+is_manual_pay = true
+delete_if_exists = false
+KillBillClient::Model::Invoice.upload_invoice_template(invoice_template, 
+                                                       is_manual_pay, 
+                                                       delete_if_exists, 
+                                                       user, 
+                                                       reason, 
+                                                       comment, 
+                                                       options)
 ```
+
 ```python
-"<meta charset=\"UTF-8\">
+invoiceApi = killbill.api.InvoiceApi()
+body = 'Some_HTML_String'
+
+invoiceApi.upload_invoice_mp_template(body,
+                                      created_by,
+                                      api_key,
+                                      api_secret)
+```
+
+> Example Response:
+
+```ruby
+"
```

Comment:
is Ruby response included here?

Reply:
Reshma, for all these template ones, example https://killbill.github.io/slate/index.html?ruby#invoice-upload-the-invoice-template-for-the-tenant only the Ruby one had a response. Should I change it to Json?

Reply:
Yes, please remove all Ruby responses. Also, as discussed on slack, if an endpoint returns a Location header, please remove the `Example Response` and ensure that the `Response` section specifies that a 201/Location header is returned.